backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -4179,4 +4179,18 @@ databaseChangeLog:
             columnName: device_specimen_type_id
         - addNotNullConstraint:
             tableName: test_order
-            columnName: device_specimen_type_id
\ No newline at end of file
+            columnName: device_specimen_type_id
+  - changeSet:
+      id: add-test-performed-code-column-to-device-supported-disease-table
+      author: boban@skylight.digital
+      comment: Add a test performed code column to store the specific LOINC code between a device and disease.
+      changes:
+        - tagDatabase:
+            tag: add-test-performed-code-column-to-device-supported-disease-table
+        - addColumn:
+            tableName: device_supported_disease
+            columns:
+              - column:
+                  name: test_performed_code
+                  type: text
+                  remarks: The code that identifies which disease was tested.
```

Comment:
Like we talked about offline, please double check that we don't need to add any no-phi permissions for this column to be viewable in Metabase 👍

Reply:
Deployed in dev3 and it is accessible!
![image](https://user-images.githubusercontent.com/10108172/211651422-d66f82c2-7967-4d4f-b622-508037ade10d.png)